ksql-engine/src/main/java/io/confluent/ksql/function/udf/geo/GeoDistance.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ **/
+
+package io.confluent.ksql.function.udf.geo;
+
+import io.confluent.ksql.function.KsqlFunctionException;
+import io.confluent.ksql.function.udf.Kudf;
+
+/**
+ * Compute the distance between two points on the surface of the earth, according to the Haversine
+ * formula for "great circle distance". The 2 input points should be specified as (lat, lon) pairs,
+ * measured in decimal degrees.
+ * 
+ * <p>An optional fifth parameter allows to specify either "MI" (miles) or "KM" (kilometers) as the
+ * desired unit for the output measurement. Default is KM.
+ *
+ */
+public class GeoDistance implements Kudf {
```

Comment:
I think that, by our convention, `GeoDistance` must be named `GeoDistanceKudf`.

Reply:
changed as suggested